met-web/src/components/publicDashboard/SubmissionTrend/SubmissionTrend.tsx

```diff
@@ -155,12 +171,18 @@ const SubmissionTrend = ({ engagement, engagementIsLoading }: SubmissionTrendPro
                             </Grid>
 
                             <Grid container item xs={8} justifyContent="center" alignItems="center">
-                                <PrimaryButton sx={{ width: isSmallScreen ? '50%' : '100%' }} onClick={clearDates}>
+                                <PrimaryButton sx={{ width: '100%', maxHeight: '34px' }} onClick={clearDates}>
```

Comment:
We should reconsider when we change style of an atomic component. Like PrimaryButton is an atomic component and here we are changing it's height. It's style should be consistent throughout the whole app and also there is a BC gov standard so we need to check with MA. What else can we do?

Reply:
Sure thing I did this because it was asked for in this task https://github.com/bcgov/met-public/issues/1536 we could revert this change need to talk to MA

Reply:
Is there a better way we can have this change rather than injecting maxHeight every time we need it?